elf/table.go

```diff
@@ -131,3 +140,33 @@ func (b *Module) DeleteElement(mp *Map, key unsafe.Pointer) error {
 
 	return nil
 }
+
+// LookupNextElement lookups the next element in the mp.
+// The key and the value are stored in the key and value unsafe.Pointer.
```

Comment:
Should this be "The key and the value are stored in the nextKey and value unsafe.Pointer."?

Reply:
@schu Thanks for reviewing. You're right "nextKey". I will try to introduce tests for that. I'll try to add test for other operations too as it seems there is currently no test.